### PR TITLE
[meson] Turn benchmark to a feature

### PR DESCRIPTION
**Pull request by ebraminio**
https://github.com/harfbuzz/harfbuzz/pull/2506#issuecomment-650814046

### CODE REVIEW

**Comment by behdad** [comment]
Thanks.

File: Makefile.am [hunk 1]
```diff
@@ -26,6 +26,12 @@ EXTRA_DIST = \
 	subprojects/libffi.wrap \
 	subprojects/proxy-libintl.wrap \
 	subprojects/zlib.wrap \
+	subprojects/google-benchmark.wrap \
+	perf/meson.build \
+	perf/perf-draw.hh \
+	perf/perf-extents.hh \
+	perf/perf-shaping.hh \
+	perf/perf.cc \
```
Comment (ebraminio) [inline Makefile.am:34]: not happy on putting perf on dist, as there it is hard to check we in autotools dist in meson, yet, isn't that but undecided.

Comment (ebraminio) [reply Makefile.am:34]: so may is ready to merge if you see the change in the right direction.

**Comment by behdad** [comment]
I'm fine as long as autotools builds fine and ships in tarball what's needed, even if can only be enabled using meson.

**Pull request merged**